### ISSUE TITLE
Usage of "effective domain" seems wrong

### ISSUE THREAD

**Issue by annevk** [w3c/webauthn#2205]
No other specification really ought to use "effective domain". That's only for `document.domain`-related business. I suspect you just want to grab an origin's host and ignore this operation.

**Comment by nicksteele** [comment]
Why is it only for `document.domain`-related business? Unclear why we can't use effective domain.

**Comment by annevk** [comment]
Because the domain field is only set when `document.domain` is used. Generally that's an internal field for the HTML Standard.

**Comment by agl** [comment]
Thanks for that. I think from the [definition of effective domain] we want to skip the second step, yes? Is there a standard name for "the origin's host or null if the origin is opaque"?

**Comment by annevk** [comment]
No, but your algorithm already returns early for an opaque origin in the prior step.
